appdata_oars.patch

```diff
@@ -0,0 +1,10 @@
+--- a/data/org.gnome.Epiphany.appdata.xml.in.in	2020-03-27 19:08:36.106873000 +0100
++++ b/data/org.gnome.Epiphany.appdata.xml.in.in	2020-05-08 22:15:41.215513649 +0200
+@@ -43,6 +43,7 @@
+   <provides>
+     <id>epiphany.desktop</id>
+   </provides>
++  <content_rating type="oars-1.0" />
```

Comment:
```suggestion
+  <content_rating type="oars-1.1" />
```

Reply:
It's just one line that's needed? Feel free to commit it upstream.